akka-actor/src/main/resources/reference.conf

```diff
@@ -1061,6 +1061,17 @@ akka {
         # How often to sweep out expired cache entries.
         # Note that this interval has nothing to do with TTLs
         cache-cleanup-interval = 120s
+
+        # Configures the list of search domains.
+        # Defaults to a system dependent lookup (on Unix like OSes, will attempt to parse /etc/resolv.conf, on
+        # other platforms, will not make any attempt to lookup the search domains). Set to a single domain, or
+        # a list of domains, eg, [ "example.com", "example.net" ].
+        search-domains = default
+
+        # Configures the number of dots that should be in a domain before a lookup will be done.
+        # Defaults to a system dependent lookup (on Unix like OSes, will attempt to parse /etc/resolv.conf, on
+        # other platforms, will default to 1).
+        ndots = default
```

Comment:
TIL `ndots` comes from the `/etc/resolv.conf`. It'd be great to mention that (with a link to https://linux.die.net/man/5/resolver).

My first impression when I saw this setting was: "Hmm, I think this name is wrong, maybe `num-of-dots` or s/thing?" but it makes sense to me now.